test/IdentityServer4.RavenDB.Storage.Tests/IdentityServer4.RavenDB.Storage.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>IdentityServer4.RavenDB.Storage.Tests</RootNamespace>
```

Comment:
I'm not sure if this line is needed - it's the same as the project name.

Reply:
👍